docs/tutorial/new_controller.rst

```diff
@@ -0,0 +1,106 @@
+
+Implementing a Controller
+-------------------------
+
+A controller closes the loop, a class derived from the abstract class
```

Comment:
Better: To close the loop a controller has to be added. This can easily be done by deriving from the  :py:class:`~pymoskito.simulation_modules.Controller` class.